src/deploy.js

```diff
@@ -65,21 +123,56 @@ const deploy = async (opts) => {
       // See: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property
     },
   };
+  const progressBar = new cliProgress.SingleBar(
+    {},
+    cliProgress.Presets.shades_classic,
+  );
   const uploader = client.uploadDir(params);
-  uploader.on('error', function (err) {
+  uploader.on('error', (err) => {
     console.error('unable to sync:', err.stack);
   });
-  uploader.on('progress', function () {
-    console.log('progress', uploader.progressAmount, uploader.progressTotal);
+  uploader.on('progress', () => {
+    progressBar.start(uploader.progressTotal, 0);
+    progressBar.update(uploader.progressAmount);
   });
-  uploader.on('end', function () {
-    console.log('done uploading');
+  uploader.on('end', () => {
+    progressBar.stop();
+    // TODO: insert here code that should be executed once the upload is done
```

Comment:
This should be done. You can always create helper functions.

Reply:
@juancarlosfarah Yes, you are right. But I would require more time to do this. Might consider implementing this when finalizing the `deploy` command.